rubicon_ml/viz/experiments_table.py

```diff
@@ -135,6 +148,29 @@ def layout(self):
             label="toggle columns",
         )
 
+        publish_modal = dbc.Modal(
+            [
+                dbc.ModalHeader(
+                    dbc.ModalTitle("publish selected experiments"),
+                    close_button=True,
+                ),
+                dbc.ModalBody(
+                    [
+                        dbc.Label("catalog YAML output path"),
+                        dbc.Input(
+                            id="publish-path-input",
+                            type="text",
+                            value=os.path.join(os.getcwd(), "rubicon-ml-catalog.yml"),
```

Comment:
is it possible to have the user define the name of the catalog file?

Reply:
you can change the text in the modal that pops up to change the path. is there some kind of messaging I should add to make that more obvious?

Reply:
Since it was gray I assumed it was grayed out - could be just me. To be safe we could try doing something like `eneter catalog YAML output path here`. Whatever you decide works.

Reply:
gotcha - I updated the text and also made the text box bigger so it can fully show longer paths

Reply:
awesome thanks!